docs/user/README.md

```diff
@@ -2,25 +2,46 @@
 
 ## What Is API Gateway?
 
-API Gateway provides functionalities that allow you to expose and secure APIs by using [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper) and the [Istio Service Mesh](https://istio.io/) resources.
+API Gateway is a Kyma module, which provides functionalities that allow you to expose and secure APIs by using [Ory Oathkeeper](https://www.ory.sh/docs/oathkeeper) and [Istio](https://istio.io/) resources.
 
-## Kyma API Gateway Operator
+By default, the API Gateway module is automatically added once you create a Kyma runtime instance. To use the API Gateway module, the Istio module must be also added.
```

Comment:
You can mention that Istio, which API Gateway is dependant on is also enabled by default

Reply:
sure, great idea